DS/DS2/InfixToPostfix.java

```diff
@@ -0,0 +1,132 @@
+import java.util.Scanner;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+class Stack {
+	Object listInformation;
+	char listInformationInChar;
+	Stack addressOfNextNode,top;
+	public Stack() {
+		top=null;
+	}
+	public void pushAnItem(Object listInformation) {
+		Stack node=new Stack();
+		node.listInformation=listInformation;
+		if(top==null) {
+			top=node;
+			node.addressOfNextNode=null;
+		}
+		else {
+			node.addressOfNextNode=top;
+			top=node;
+		}
+	}
+	public void pushAnItem(char listInformationInChar) {
+		Stack node=new Stack();
+		node.listInformationInChar=listInformationInChar;
+		if(top==null) {
+			top=node;
+			node.addressOfNextNode=null;
+		}
+		else {
+			node.addressOfNextNode=top;
+			top=node;
+		}
+	}
+	public Stack popAnItem() {
+		Stack stackTopElement;
+		if(top==null) {
+			stackTopElement=null;
+			return stackTopElement;
+		}
+		else if(top.addressOfNextNode==null) {
+			stackTopElement=top;
+			top=null;
+			return stackTopElement;
+		}
+		else {
+			stackTopElement=top;
+			top=top.addressOfNextNode;
+			return stackTopElement;
+		}
+	}
+	public void display() {
+		if(top==null) {
+			System.out.println("Stack is empty");
+		}
+		else {
+			Stack traversingNode=top;
+			while(traversingNode!=null) {
+				System.out.println(traversingNode.listInformation);
+				traversingNode=traversingNode.addressOfNextNode;
+			}
+		}
+	}
+}
+class Precedence {
+	public int getPrcedence(char operand) {
+		int precendendeNumber=0;
+		switch(operand) {
+		case '+':precendendeNumber=1;
+			break;
+		case '-':precendendeNumber=1;
+			break;
+		case '*':precendendeNumber=2;
+			break;
+		case '/':precendendeNumber=2;
+			break;
+		case '%':precendendeNumber=3;
+			break;
+		case ')':precendendeNumber=0;
+			break;
+		case '(':precendendeNumber=0;
+		break;
+		}
+		return precendendeNumber;
+	}
+}
+public class InfixToPostfix extends StackUsingLinkedList{
```

Comment:
ACTION : Why are you extending from StackUsingLinkedList, we just to create instance of StackUsingLinkedList class. We don't have to extend to create a instance. I